src/main/resources/data/forge/tags/items/crops.json

```diff
@@ -0,0 +1,14 @@
+{
+    "values": [
+        "#forge:crops/bellpepper",
+        "#forge:crops/cabbage",
+        "#forge:crops/chickpea",
+        "#forge:crops/durumwheat",
+        "#forge:crops/eggplant",
+        "#forge:crops/garlic",
+        "#forge:crops/onion",
+        "#forge:crops/rice",
+        "#forge:crops/soybean",
+        "#forge:crops/tomato"

```

Comment:
we shouldnt add them to the crops tag, because they're not behaving like normal crops which other mods would expect